EasyPost.Tests/ServicesTests/WithParameters/ReferralCustomerServiceTest.cs

```diff
@@ -62,26 +61,5 @@ public async Task TestAll()
         #endregion
 
         #endregion
-
-        private static string ReferralCustomerKey => TestUtils.GetApiKey(TestUtils.ApiKey.Referral);
```

Comment:
**Author's Note** This helper function is not needed in the unit tests for parameters, as it's handled inside the fixtures